### ISSUE TITLE
The command key word font colour is really hard to read

### ISSUE THREAD

**Issue by sonichanxiao** [workshopper/learnyounode#493]
I guess I am not the first one who has this problem.

Any solution or workaround for this?

**Comment by MattTunny** [comment]
yeah i had the same issue, just go into terminal > preferences > profiles and change the ANSI colors or a new theme
